packages/app/auth/hooks/useLogin.ts

```diff
@@ -8,20 +8,13 @@ interface UserForm {
 }
 
 interface UseLoginReturn {
-  form: UseFormReturn<UserForm>;
   handleLogin: (data: UserForm) => void;
   hasError: boolean;
 }
 
 export const useLogin = (): UseLoginReturn => {
   const { mutateAsync: signIn, error } = queryTrpc.signIn.useMutation();
   const sessionSignIn = useSessionSignIn();
-  const form = useForm({
```

Comment:
do we lose defaultValues going this route? sorry if i missed where you moved them to

Reply:
It works perfectly even if you don't add default values, so I thought don't add them if we don't need them